cypress/support/gutenberg/commands/controls.js

```diff
@@ -412,6 +415,100 @@ function radioControl( name, value, options = {} ) {
 		.click( { force: true } )
 }
 
+/**
+ * Command for adjusting the form token input.
+ *
+ * @param {string} name
+ * @param {*} value
+ * @param {Object} options
+ */
+function formTokenControl( name, value, options = {} ) {
+	const {
+		isInPopover = false,
+		beforeAdjust = () => {},
+		parentSelector,
+		supportedDelimiter = [],
+	} = options
+
+	const selector = () => cy.getBaseControl( name, {
+		isInPopover,
+		parentSelector,
+		supportedDelimiter,
+	} )
+
+	const isWpLessThan58 = compareVersions( Cypress.env( 'WORDPRESS_VERSION' ), '5.8.0', '<' )
+
+	beforeAdjust( name, value, options )
+	value.forEach( val => {
+		if ( isWpLessThan58 ) {
+		// The suggestions list does not display on WP versions less than 5.8. Type and enter the values.
```

Comment:
This is false. Form token fields are designed to have suggestion list.

Reply:
I meant when the input is initially clicked. `__experimentalExpandOnFocus` only works on WP 5.8

Reply:
will update the comment